db/schema.rb

```diff
@@ -90,8 +90,8 @@
   end
 
   create_table "users", force: :cascade do |t|
-    t.string   "email",                  limit: 255, default: "", null: false
-    t.string   "encrypted_password",     limit: 255, default: "", null: false
+    t.string   "email",                  limit: 255, default: "",   null: false
```

Comment:
Weird ... Any idea why this is showing up as an addition? Seems as though users have always had emails and passwords.